Composer/packages/client/src/pages/design/exportSkillModal/content/SelectProfile.tsx

```diff
@@ -173,38 +208,49 @@ export const SelectProfile: React.FC<ContentProps> = ({
     }
   }, [id]);
 
-  return isProfileValid ? (
+  const handleShowCreateProfileDialog = () => {
+    setShowCreateProfileDialog(true);
+  };
+
+  return !showCreateProfileDialog ? (
     <div css={styles.container}>
       <Dropdown
         required
         ariaLabel={formatMessage('Select a publishing profile')}
         label={formatMessage('Publishing Profile')}
         options={publishingOptions}
         placeholder={formatMessage('Select one')}
-        selectedKey={publishingOptions[0].key}
+        selectedKey={selectedKey}
         styles={{ root: { paddingBottom: '8px' } }}
         onChange={handleCurrentProfileChange}
+        onRenderTitle={onRenderTitle}
```

Comment:
Q: this is a custom renderer for selected option, but you are always rendering the first item if exist, is that intentional?
https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown

Reply:
Have checked with the example in fluentui website. Rendering the first item is the default render,  renderer will render the selected option if the selected option changed

Reply:
ah, the input arg is the list of selected items and this dropdown is single selection. I get it.